libs/constrained-generators/src/Constrained/Base.hs

```diff
@@ -648,9 +657,9 @@ instance (HasSpec fn a, Arbitrary (TypeSpec fn a)) => Arbitrary (Specification f
               TypeSpec <$> arbitrary <*> vectorOf len (genFromSpec @fn TrueSpec)
           )
         , (1, ErrorSpec <$> arbitrary)
-        , (1, pure $ MemberSpec [])
-        , -- Recurse to make sure we apply the tricks for generating suspended specs multiple times
-          (1, arbitrary)
+        , --  , (1, pure $ MemberSpec []) -- This is always inconsistent, So I have temporarily remove it subject to discussion T.S.
+          -- Recurse to make sure we apply the tricks for generating suspended specs multiple times
+          (4, arbitrary)
```

Comment:
We probably want to undo these changes.

Reply:
I guess since this is an arbitrary instance. We want it to generate insconsistent things once and a while. I added it back